internal/controller/linodevpc_controller.go

```diff
@@ -298,7 +294,7 @@ func (r *LinodeVPCReconciler) reconcileDelete(ctx context.Context, logger logr.L
 		logger.Info("VPC ID is missing, nothing to do")
 	}
 
-	conditions.MarkFalse(vpcScope.LinodeVPC, clusterv1.ReadyCondition, clusterv1.DeletedReason, clusterv1.ConditionSeverityInfo, "VPC deleted")
+	conditions.MarkFalse(vpcScope.LinodeVPC, clusterv1.ReadyCondition, clusterv1.DeletedReason, "", "VPC deleted")
```

Comment:
Do we want to keep this consistent with other code? Like we are doing `conditions.MarkFalse(X, Y, Z, "", "%s", "Error msg or empty")`. I am ok with any approach. I just saw that you had `"%s", ""` at all places previously, hence was checking.

Reply:
Hmmm it works either way but for the sake of consistency I'll switch to using `"%s", ""`